api/app.js

```diff
@@ -14,6 +15,15 @@ module.exports = {
         app.post('/sendCode', authController.sendCode);
         app.post('/sendcall', authController.sendCall);
         app.post('/signup', authController.signup);
+        app.post('/get-groups', groupAdminController.getGroups(mongoCollections));        
+        app.post('/get-dashboard-stats', groupAdminController.getDashboardStats(mongoCollections));
+        app.post('/get-messages-yearly-count', groupAdminController.getMessagesYearlyCount(mongoCollections));
```

Comment:
It is something wrong here about REST architecture.
Firstly, you should not use verbs ("get") in routes.
Secondly, I see verb get in url for post request. Looks like a confusion.